webpage/src/app/profile/page.tsx

```diff
@@ -2,7 +2,7 @@
 
 import { useEffect, useState } from 'react';
 
-import { useGetBasicUserIngormation } from '../../hooks/use-get-user-basic-information.hook';
+import { useGetBasicUserIngormation as useGetBasicUserInformation } from '../../hooks/use-get-user-basic-information.hook';
```

Comment:
I think that here you can rename the function instead of renaming it in the file